db/models.py

```diff
@@ -24,6 +24,6 @@ class Player(models.Model):
     nickname = models.CharField(max_length=255, unique=True)
     email = models.EmailField(max_length=255)
     bio = models.CharField(max_length=255)
-    race = models.ForeignKey(Race, on_delete=models.CASCADE)
-    guild = models.ForeignKey(Guild, on_delete=models.CASCADE, null=True)
+    race = models.ForeignKey(Race, on_delete=models.SET_NULL)
```

Comment:
Race shouldn't actually have `SET_NULL` option. Only guild.

Reply:
It depends on your realization. I don't want to lose all information about the player if the race will be deleted

Reply:
Then you should use `PROTECT`.